DigitalLearningSolutions.Web/Controllers/ForgotPasswordController.cs

```diff
@@ -37,14 +40,15 @@ public IActionResult Index(string? emailAddress)
             {
                 passwordResetService.GenerateAndSendPasswordResetLink
                 (
-                    emailAddress,
+                    model.EmailAddress,
                     baseUrl
                 );
             }
             catch (EmailAddressNotFoundException)
             {
-                ForgotPasswordViewModel model = GetEmailNotFoundErrorModel(emailAddress);
-                return Index(model);
+                ModelState.AddModelError("EmailAddress","User with this email address does not exist. Please try again.");
```

Comment:
I think this will need a space between , and " - probably worth running the formatter quickly.